### DISCUSSION TITLE
在线预览docx、ppt文档失败

### DISCUSSION THREAD

**Discussion by weisbergs** [AlistGo/alist#5596] [Invalid question]
<img width="1324" alt="image" src="https://github.com/alist-org/alist/assets/62733674/548faaf1-0864-4ba8-8d57-dd45dbf887d5">

**Comment by Sanmejie** [comment]
![image](https://github.com/alist-org/alist/assets/40269911/bd8512af-aebb-4482-85f2-5daeebf584c5)
我也遇到同样的问题。
看alist可以不可以支持集成下KKFileView
相关讨论https://github.com/alist-org/alist/discussions/3908

**Reply by Sanmejie** [reply]
https://github.com/alist-org/alist/discussions/594 
找到原因了， 我端口不是443 或80。 公司内部署的我现在只能有非这俩的端口搭建alist。。。

**Reply by weisbergs** [reply]
我是用https，端口用的还是5244，改成443就可以吗？
